### PR TITLE
record inclusion: use system identity to accept inclusion request when can_include_directly

### PR DESCRIPTION
**Pull request by anikachurilova**
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/176

Request gets auto-submitted by system user
Screenshot 2024-07-26 at 10 18 23

### CODE REVIEW

File: invenio_rdm_records/services/community_inclusion/service.py [hunk 1]
```diff
@@ -64,7 +65,7 @@ def include(self, identity, community, request, uow):
 
         if can_include_directly:
             request_item = current_requests_service.execute_action(
-                identity, request.id, "accept", data=None, uow=uow
+                system_identity, request.id, "accept", data=None, uow=uow
```
Comment (kpsherva) [inline invenio_rdm_records/services/community_inclusion/service.py:68]: 👀 to discuss if we want to change it

**Review by jrcastro2 — APPROVED** [review]
LGTM! 🚀

File: invenio_rdm_records/services/community_inclusion/service.py (hunk 1 above)
Comment (anikachurilova) [reply invenio_rdm_records/services/community_inclusion/service.py:69]: Discussed irl, see the changes pushed!

File: invenio_rdm_records/services/community_inclusion/service.py [hunk 2]
```diff
@@ -64,7 +66,23 @@ def include(self, identity, community, request, uow):
...
+            )
+
+            data = {
+                "payload": {
+                    "content": "This draft review request was automatically created and accepted because "
```
Comment (ntarocco) [inline invenio_rdm_records/services/community_inclusion/service.py:74]: This sentence should be translatable

File: invenio_rdm_records/services/community_inclusion/service.py [hunk 3]
```diff
@@ -64,7 +66,23 @@ def include(self, identity, community, request, uow):
...
+                    "the record was published directly to community, bypassing the review "
+                    "process."
```
Comment (ntarocco) [inline invenio_rdm_records/services/community_inclusion/service.py:76]: Shall we make it more simple? What do you think of this?

`This request has been automatically accepted, as the uploader can submit directly without review.`

**Review by ntarocco — APPROVED** [review]

**Pull request merged**